packages/orm/objection/src/decorators/belongsToOne.ts

```
@@ -1,14 +1,55 @@
  * @objection
  */
-export function BelongsToOne(opts?: RelationshipOptsWithoutThrough): PropertyDecorator {
-  return RelatesTo(Model.BelongsToOneRelation, opts);
+export function BelongsToOne(
+  typeOrOpts?: ModelClassSpecifier | RelationshipOptsWithoutThrough,
+  opts?: RelationshipOptsWithoutThrough
+): PropertyDecorator {
+  // For backwards compatibility
+  if (!typeOrOpts && !opts) return RelatesTo(Model.BelongsToOneRelation, undefined);
+  if (!typeOrOpts && opts) {
+    typeOrOpts = opts;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)